Sources/SwifterSwift/SwiftStdlib/KeyedDecodingContainer.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  KeyedDecodingContainer.swift
+//  SwifterSwift
+//
+//  Created by Francesco Deliro on 23/10/2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+#if canImport(Foundation)
+import Foundation
+#endif
+
+public extension KeyedDecodingContainer where Key: CodingKey {
```

Comment:
`Key` already conforms to `CodingKey`.
```suggestion
public extension KeyedDecodingContainer {
```